protos/flyteidl/admin/project.proto

```diff
@@ -29,6 +29,8 @@ message Project {
     // Leverage Labels from flyteidel.admin.common.proto to
     // tag projects with ownership information.
     Labels labels = 5;
+
+    bool archived = 6;
```

Comment:
what do you think on adding a state enum like https://github.com/lyft/flyteidl/blob/c3baba8983019680ef57b6244cea36ba951233ed/protos/flyteidl/admin/common.proto#L25? not sure if overkill for entities like project but even for sandbox deploys could be nice to one day have a system-generated type attribute too

Reply:
Yeah makes sense. 👍 Didn't know this existed. Thanks for pointing out.

Reply:
Does it make sense to use that enum directly than creating a new one? What is the semantics or usage of that enum?

Reply:
Hmm probably not according to the comment of NamedEntity.

Reply:
yeah, maybe better to use a new enum definition in this case.